app/workers/certificates.php

```diff
@@ -375,18 +377,38 @@ private function notifyError(string $domain, string $errorMessage, int $attempt)
         Console::warning('Cannot renew domain (' . $domain . ') on attempt no. ' . $attempt . ' certificate: ' . $errorMessage);
 
         // Send mail to administratore mail
+
+        $locale = new Locale(App::getEnv('_APP_LOCALE', 'en'));
+        if (!$locale->getText('emails.sender') || !$locale->getText("emails.certificate.hello") || !$locale->getText("emails.certificate.subject") || !$locale->getText("emails.certificate.body") || !$locale->getText("emails.certificate.footer") || !$locale->getText("emails.certificate.thanks") || !$locale->getText("emails.certificate.signature")) {
+            $locale->setDefault('en');
+        }
+
+        $body = Template::fromFile(__DIR__ . '/../config/locale/templates/email-base.tpl');
+
+                $subject = \sprintf($locale->getText("emails.certificate.subject"), $domain);
+                $body->setParam('{{domain}}', $domain);
+                $body->setParam('{{error}}', $errorMessage);
+                $body->setParam('{{attempt}}', $attempt);
```

Comment:
```suggestion
            $subject = \sprintf($locale->getText("emails.certificate.subject"), $domain);
            $body->setParam('{{domain}}', $domain);
            $body->setParam('{{error}}', $errorMessage);
            $body->setParam('{{attempt}}', $attempt);
```

Indentation is wrong.